### ISSUE TITLE
Can't play video from custom UIView

### ISSUE THREAD

**Issue by hangocanh2303** [rinov/YoutubeKit#5]
I can't play youtube video from custom UIView, but video can play in self.view of UIViewController. Why?

**Comment by hangocanh2303** [comment]
@rinov You can solve this problem for me ? I really want to use your library for my app.

**Comment by Cortlandd** [comment]
@hangocanh2303 Did you ever figure this out?

**Comment by hangocanh2303** [comment]
@Cortlandd when i place customUIView = player in my viewController, video can't play. I have not solved it yet. have you any solution?

**Comment by Cortlandd** [comment]
@hangocanh2303 I did the same with no luck. Even changed view for player to ```@IBOutlet var playerview: YTSwiftyPlayer!``` then tried to reference it in viewdidload with no luck.

**Comment by Cortlandd** [comment]
@hangocanh2303 fixed

```swift
var playerView: YTSwiftyPlayer!

    override func viewDidLoad() {
        super.viewDidLoad()
        
        // Create a new player
        playerView = YTSwiftyPlayer(frame: CGRect(x: 0, y: 0, width: view.bounds.width, height: 300), playerVars: [.autoplay(true), .videoID("V1EYgeTER_I")])
        
        view.addSubview(playerView)
        
        // Set delegate for detect callback information from the player.
        playerView.delegate = self as YTSwiftyPlayerDelegate
        
        // Load the video.
        playerView.loadPlayer()
        
        // Do any additional setup after loading the view, typically from a nib.
        
    }
```

**Comment by hangocanh2303** [comment]
oh, many thanks @Cortlandd .

**Comment by hangocanh2303** [comment]
@Cortlandd in your code, YTSwiftyPlayerDelegate working?

**Comment by Cortlandd** [comment]
@hangocanh2303 haven't had to use them yet.

**Comment by rinov** [comment]
Hey guys,
Delegate method issue has been fixed in v0.2.2, Please update pods.

```
$ pod repo update
$ pod install
``` 

Thank you.

FYI: https://github.com/rinov/YoutubeKit/issues/8
